scripts/bench_python_cpp_algos.py

```diff
@@ -388,8 +417,49 @@ def load_graph(graph_path, edge_properties=None):
         graph = load_graph(graph_path, [])
 
         if args.application == "pagerank":
-            for _ in range(args.trials):
-                run_pagerank(graph, input)
+            data = run_routine(run_pagerank, data, args.trials, (graph, input))
+
+    elif args.application in ["all"]:
+        graph_path = f"{args.input_dir}/{input['transpose_input']}"
+        if not os.path.exists(graph_path):
+            print(f"Symmetric Graph doesn't exist: {graph_path}")
+
+        graph_path = f"{args.input_dir}/{input['name']}"
+        if not os.path.exists(graph_path):
+            print(f"Graph doesn't exist: {graph_path}")
+
+        graph = load_graph(graph_path)
+
+        data = run_routine(run_bfs, data, args.trials, (graph, input, args.source_nodes))
+        data = run_routine(run_sssp, data, args.trials, (graph, input, args.source_nodes))
+        data = run_routine(run_jaccard, data, args.trials, (graph, input))
+        data = run_routine(run_bc, data, args.trials, (graph, input, args.source_nodes, 4))
+
+        graph_path = f"{args.input_dir}/{input['symmetric_clean_input']}"
+        if not os.path.exists(graph_path):
+            print(f"Symmetric clean Graph doesn't exist: {graph_path}")
+
+        graph = load_graph(graph_path, [])
+
+        data = run_routine(run_tc, data, args.trials, (graph, input))
+        data = run_routine(run_cc, data, args.trials, (graph, input))
+        data = run_routine(run_kcore, data, args.trials, (graph, input))
+
+        graph_path = f"{args.input_dir}/{input['symmetric_input']}"
+        if not os.path.exists(graph_path):
+            print(f"Symmetric Graph doesn't exist: {graph_path}")
+
+        graph = load_graph(graph_path)
```

Comment:
Minor optimization: Louvain uses the same graph as tc, cc, and kcore with the only difference being it loads the edge property (which should not be the required thing anymore but we should test with the edge property for these graphs). So, if we can load the edge property to the existing graph, that will save some loading time for bigger graphs.